kythe/rust/cargo/BUILD

```diff
@@ -43,6 +43,11 @@ alias(
     actual = "@raze__quick_error__1_2_3//:quick_error",
     tags = ["cargo-raze"],
 )
+alias(
```

Comment:
Note, I will skip checking BUILD file changes and what seem to be automated edits.

I really recommend making cargo raze changes separate pull requests.

Reply:
I agree, I'll do that in the future